galaxy/values-cvmfs.yaml

```diff
@@ -33,6 +33,40 @@ persistence:
   size: 10Gi
   mountPath: /galaxy/server/database
 
+#initContainers:
```

Comment:
We should probably call this extraInitContainers: https://github.com/helm/charts/search?q=extraInitContainers&unscoped_q=extraInitContainers

Reply:
I agree. Will make the change.

Reply:
Changed!